.travis.yml

```diff
@@ -85,6 +85,7 @@ script:
       else
         py.test -m "not avoid_travis" distributed $PYTEST_OPTIONS;
       fi;
+    - flake8 distributed/*.py distributed/{bokeh,protocol,deploy}/*.py # no tests yet
```

Comment:
Is there a way to have it run over everything *except* tests, instead of manually listing subpackages?